src/tools/miri/miri-script/src/commands.rs

```diff
@@ -173,7 +173,7 @@ impl Command {
         // the merge has confused the heck out of josh in the past.
         // We pass `--no-verify` to avoid running git hooks like `./miri fmt` that could in turn
         // trigger auto-actions.
-        sh.write_file("rust-version", &commit)?;
+        sh.write_file("rust-version", format!("{commit}\n"))?;
```

Comment:
Something went wrong here, I don't think this PR should change Miri. Looks like a rebase gone wrong, the diff shown here has already landed on master.

Reply:
I've rebased on top of master, but it looks like a change was made that broke `env`. Let me fix that and push a change.